### DISCUSSION TITLE
'.quasar/client-entry.js' gets overwritten by last server start and previous server picks its configuration

### DISCUSSION THREAD

**Discussion by CamilleDrapier** [quasarframework/quasar#15956] [CLI - PWA mode] [answered]
I'm not sure what I'm trying to do is supposed to be supported, if that is not the case feel free to let me know. 

It happens quite often that I run 2 instances of Quasar (accessible on different ports): one for regular development checking, and one for running integration tests in parallel  (sometimes). They are run from the same directory but in different terminals. Those two applications are configured slightly differently with which boot/CSS files they are using through an ENV variable injected to the quasar command and used in the quasar configuration file (`quasar.conf.js`) such as this: 

```
  css: [
    process.env.TEST ? 'no-animation.scss' : '',
  ],
```

This CSS file contains for example a bit of CSS to disable all animations. 

When I start my dev server, everything looks good, but when I start running the integration server and go back to interact with my dev application it does not behave the normal way but behaves the same way as my integration application, which is a bit disturbing. As mentioned in the title, this probably happens because the `.quasar/client-entry.js` file that apparently caches part of the result of running the `quasar.conf.js` file is overwritten by the last server start but any server that is already started would update itself with the latest version of that cached file?

There are workarounds such as running those two servers in two different directories (two separate checkouts of the same repo), but it is not very practical. Another thing I can think of is Pass the env variable to the application and then use it to conditionally inject CSS; probably the same can be done to run things or not in boot files that would be always loaded. 

Is this behavior from Quasar the expected behavior? If yes, then are there some recommended ways to work around the issue I'm facing?

**Comment by rstoenescu** [comment, marked as answer]
We currently do not support more than one instance of Quasar CLI running on the same project.
This will be supported in the future.

The env variables can be passed to quasar.config.js
